app/formats/document_types.yml

```diff
@@ -43,6 +45,10 @@
       label: World locations
       type: multi_association
       document_type: world_location
+    - id: worldwide_organisations
```

Comment:
They changed because I updated the order based on the content document. `id`s use a mix of `_` and `-` for example `any-whitehall-publication`, `any-mainstream-publication`, `travel_advice`, `case_study`. From a frontend perspective I prefer using `-` unless this is being used elsewhere and we need it to be with underscore.

Reply:
I'm OK with that, but it's now inconsistent with the rest of the file. Can you move everything one way or the other?

Reply:
Sure, I'll do that.